docs/gallery.md

```diff
@@ -0,0 +1,110 @@
+# Gallery of Jupyter Books
+Jupyter Book technology has been applied across a wide range of use cases, including curricula vitae, official educational textbooks, student portfolios, lab manuals, and technical documentation.
+This is a gallery of Jupyter Books built from across the community. If you’d like to add your book to this list, simply add an entry to this gallery.yml file and open a Pull Request to add it. Note that the description should be brief, limited to 500 characters (including spaces).
+Below is a gallery showcasing examples of such outputs.
```

Comment:
I'd recommend the following steps:

- only commit the explanation version of this page, and include the "gallery" with an `{include}` directive.
- in our nox file , make sure that `nox -s docs` etc build a local file like `_build/temp/gallery.txt` _first_.
- then in `docs/gallery.md` you can do `{include} _build/temp/gallery.txt`

That way the gallery is auto-generated each time the docs update, and we don't duplicate content.